requirements-server.txt

```diff
@@ -8,7 +8,7 @@
 -r requirements.dev.txt
 
 # For worker manager
-boto3==1.9.228
+boto3~=1.14.33
```

Comment:
Can you remove the `~=` in all dependencies? We want to pin dependencies to exact versions using `==` instead.

Also, are these dependency upgrades required for the mypy checks to work? If not, we should do them in a separate PR (dependabot might have already caught a few of them).